packages/documentation/cypress/snapshots/components/card-control.snapshot.ts

```diff
@@ -0,0 +1,7 @@
+describe('CardControl', () => {
```

Comment:
Could you add tests with cypress-axe as well like for example https://github.com/swisspost/design-system/blob/eee9ee57125889bfd8ff3ef6cac5e1d4ffeb26ca/packages/components/cypress/e2e/accordion.cy.ts#L51-L56 ?

Reply:
Sure.
These tests didn't exist when I finished my PR. But we should definitely add them!